modules/nf-core/scimap/spatiallda/main.nf

```diff
@@ -0,0 +1,36 @@
+process SCIMAP_SPATIALLDA {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "docker.io/miguelib/scimap:0.0.3"
```

Comment:
Use proper container declaration, see other modules for reference. Quay (quay.io) is normally used for containers, push the container there as well if possible.

Reply:
As it's not in bioconda and there are no current plans of adding it to it we can't add it to quay.io